dhis-2/dhis-web-api/src/main/resources/openapi/AbstractCrudController.md

```diff
@@ -52,7 +52,7 @@ Combine filters with `AND` (default) or `OR`
 ### `*.parameter.ImportOptions.preheatCache`
 Turn cache-map preheating on/off. This is on by default, turning this off will make initial load time for importer much shorter (but will make the import itself slower). This is mostly used for cases where you have a small XML/JSON file you want to import, and don't want to wait for cache-map preheating.
 
-### `*.parameter.ImportOptions.mergeMode`
+### `*.parameter.ImportOptions.metadataMergeMode`
```

Comment:
This rename of the field needs to be reverted here and in the `ImportOptions` class. In general when renaming classes to add a prefix or suffix it generally is not a good idea to also rename all fields accordingly. The extra word is already in the type information so doubling it in the parameter or field name does only add clutter but now additional information. And as here it could break the API.

Reply:
will revert this

Reply:
will leave it as just class renames - will revert variable names